docs/GettingStarted.md

```diff
@@ -221,6 +221,23 @@ def finish(name, id, payload)
   end
 end
 ```
+#####Enriching traces from nested methods
+
+You can tag additional information to current active span from any method. Note however that if the method is called and there is no span currently active `active_span` will be nil. 
+
+```ruby
```

Comment:
I still think this example should be more generalized; don't want users to think this only applies to Rails controllers. See my comment from my previous review.

Reply:
Good point! Fixed